client/components/Editor/utils/codeHighlightStyle.ts

```diff
@@ -0,0 +1,19 @@
+import { defaultHighlightStyle } from '@codemirror/language';
+
+import { DocJson } from 'types';
+
+export default (doc: DocJson): string => {
```

Comment:
I really encourage exporting functions by name so that there's a canonical name we can use to refer to them!